content/id/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Gateway API
+content_type: concept
+description: >-
+  Gateway API merupakan bagian dari API yang menyediakan penyediaan infrastruktur dinamis dan pengaturan trafik lanjutan.
+weight: 55
+---
+
+<!-- overview -->
+
+Gateway API menyediakan layanan jaringan dengan menggunakan mekanisme konfigurasi yang mudah dieksten, berorientasi _role_, dan mengerti konsep protokol. [Gateway API](https://gateway-api.sigs.k8s.io/) adalah sebuah {{<glossary_tooltip text="add-on" term_id="addons">}} yang berisi [jenis-jenis](https://gateway-api.sigs.k8s.io/references/spec/) API yang menyediakan penyediaan infrastruktur dinamis dan pengaturan trafik tingkat lanjut.
```

Comment:
`dieksten` enaknya bagaimana ya? @girikuncoro Bagaimana kalau `di-extend` saja?

Reply:
sounds good, let me fix it

Reply:
fixed